var/spack/repos/builtin/packages/chill/package.py

```diff
@@ -0,0 +1,69 @@
+##############################################################################
+# Copyright (c) 2013-2019, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#-----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+#-----------------------------------------------------------------------------
+from spack import *
+
+
+class Chill(Package):
```

Comment:
could this be changed to an `AutotoolsPackage` instead of just package ? Looks like you support Autotools or cmake, right ?

Reply:
If I remember right, there were some additional tasks that didn't fit into the AutotoolsPackage workflow.

Reply:
It should be possible to extend the `AutotoolsPackage` workflow.

Reply:
I'll take another look at it